WordPress/Resources/release_notes.txt

```diff
@@ -1,3 +1 @@
-Say hello to our shiny new in-app landing screen! It almost makes you want to look at it all day and not publish anything new. Almost.
-
-You can now block sites that you follow, as well as report a post’s author for spam, inappropriate content, or possible harm. No jokes here—stay safe out there.
+* [*] [internal] Refactored fetching posts in the Reader tab, including post related operations (i.e. like/unlike, save for latter, etc.) [#20197]
```

Comment:
I decided not to include the entry about the Reader in the WordPress raw release notes (cc2136673c1500e5043bd2c73ec10e5a847cd4e3):

> It seems it would be confusing for the WordPress users to read about a Reader changes when the feature itself is not longer available in the app.

https://github.com/wordpress-mobile/WordPress-iOS/blob/1a7a09904650652f5ff760b89f868ece4b80017d/RELEASE-NOTES.txt#L4